src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -45,36 +35,37 @@ class FinancialDisclosureSection extends React.Component {
       content = (<fieldset>
         <legend>Financial Disclosure</legend>
         <div className="input-section">
-          {notRequiredMessage}
-
+          <p>In the next 4 sections you will be asked to provide your income information from the most recent tax year. Your most recent
+          tax filing will have all the information you need in it.</p>
           <p>
-            Disclosure allows VA to accurately determine whether certain Veterans
-            will be charged copays for care and medications, their eligibility for
-            other services and enrollment. Veterans are not required to disclose
-            their financial information; however, VA is not currently enrolling new
-            applicants who decline to provide their financial information unless they
-            have other qualifying eligibility factors (i.e., a former Prisoner of
-            War, in <strong>receipt of a Purple Heart, discharged for a disability incurred or
-            aggravated in the line of duty, receiving VA service-connected disability
-            compensation, receiving VA pension, in receipt of Medicaid benefits, or a
-            recently discharged Combat Veteran (e.g., OEF/OIF/OND), who was
-            discharged within the past 5 years are eligible for enrollment without
-            disclosing their financial information)</strong> but like other Veterans may
-            provide their financial information to establish their eligibility for
-            travel assistance, cost-free medications and/or medical care for services
-            unrelated to military experience.
+            Veterans are not required to disclose their financial information; however,
+            <strong> VA is not currently enrolling new applicants who decline to provide their financial information unless
+            they have other qualifying eligibility factors (QEFs).</strong>
           </p>
 
+          <p>Qualifying Eligibility Factors(QEFs):</p>
+          <ul>
+            <li>a former Prisoner of War</li>
+            <li>in receipt of a Purple Heart</li>
+            <li>discharged for a disability incurred or aggravated in the line of duty</li>
+            <li>receiving monetary compensation for VA service-connected disability</li>
+            <li>receiving VA pension</li>
+            <li>in receipt of Medicaid benefits</li>
+            <li>a recently discharged Combat Veteran, discharged within the past 5 years</li>
+          </ul>
+
+          <p>Veterans with qualifying eligibility factors (QEFs) may provide their financial
```

Comment:
Can we phrase this in more friendly, plain language? Perhaps: "Even if one of the qualifying eligibility factors applies to you, you may still benefit from completing your financial information. Based on your financial information, you may be eligible for additional benefits, including travel assistance, cost-free medications, and medical care for services unrelated to your military experience."